packages/node/src/handlers/call-api.test.ts

```diff
@@ -31,10 +31,108 @@ describe('callApi', () => {
         parameters: { from: 'ETH' },
         securitySchemes: [
           {
-            in: 'query',
-            name: 'access_key',
             securitySchemeName: 'My Security Scheme',
-            type: 'apiKey',
+            value: 'supersecret',
+          },
+        ],
+      },
+      { timeout: 20000 }
+    );
+  });
+
+  it('calls the adapter with the given parameters (with _relay_metadata set to v1 lowercase)', async () => {
```

Comment:
There's a lot of duplication in these tests just to test different values. Can we rather use `.forEach` (like [here](https://github.com/api3dao/airnode/blob/master/packages/node/src/providers/state.test.ts#L153)) or `test.each`. I wasn't aware of `test.each` until recently which is why I didn't use it in the past, but also `.forEach` seems a bit more intuitive.

You probably need some object to loop over like:

```ts
const relayPermutations = [
  { value: 'v1', metadataExpected: true },
  { value: 'V1', metadataExpected: true },
  { value: 'false', metadataExpected: false },
  { value: undefined, metadataExpected: false },
];
```

Reply:
I have used the .forEach approach in the past but I'll dig more into the test.each too. Thanks :+1: